Cargo.toml

```diff
@@ -67,8 +67,8 @@ pin-project = "1.0"
 quick-xml = { version = "0.31", features = ["serialize", "serde-types"] }
 rand = "0.8"
 reqwest = { version = "0.12", features = [
-  "json",
-  "stream",
+    "json",
```

Comment:
These should be gone - just 2 spaces now, accordingly to `.editorconfig`.

Reply:
🤦🏻‍♀️

Reply:
Ok, I believe _now_ I've got RustRover configured right for EditorConfig... *sigh*